src/transformers/modeling_outputs.py

```diff
@@ -812,3 +812,36 @@ class Seq2SeqQuestionAnsweringModelOutput(ModelOutput):
     encoder_last_hidden_state: Optional[torch.FloatTensor] = None
     encoder_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     encoder_attentions: Optional[Tuple[torch.FloatTensor]] = None
+
+
+@dataclass
+class SemanticSegmentationModelOutput(ModelOutput):
+    """
+    Base class for outputs of instance segmentation models.
+
+    Args:
+        loss (`torch.FloatTensor` of shape `(1,)`, *optional*, returned when `labels` is provided):
+            Classification (or regression if config.num_labels==1) loss.
```

Comment:
Do we also want to tackle regression tasks with this output class?

Depth estimation is an example of per-pixel regression, but that's a different task.

Reply:
We can add it to the docstring once it's supported. For now, the models don't use `problem_type` to infer the proper loss function.